docs/development_environment.md

```diff
@@ -11,7 +11,7 @@ You'll need to set up a development environment if you want to develop a new fea
 Install the core dependencies.
 
 ```shell
-$ sudo apt-get install python3-pip python3-dev python3-venv
+$ sudo apt-get install python3-pip python3.7-dev python3.7-venv
```

Comment:
We shouldn't add this. We always want the latest Python 3 available.

Reply:
When I ran this, I didn't get the latest version. On Ubuntu it pulled the 3.6 versions, and wouldn't pull down 3.7 unless I forced it with a version number.

What are your thoughts on the Windows SLS Ubuntu, which has hard coded versions of 3.7?

Reply:
The `python3` version always points at the latest version available in the repository, unless you have added a custom repository for Python 3.7